storage-manager/src/unit_tests.cpp

```diff
@@ -348,8 +348,9 @@ void metadataJournalTest(std::size_t size, off_t offset)
     w.run();
 
     // verify response
-    err = ::recv(sessionSock, buf, 1024, MSG_DONTWAIT);
-    sm_response *resp = (sm_response *) buf;
+    uint8_t bufRead[1024];
+    err = ::recv(sessionSock, bufRead, 1024, MSG_DONTWAIT);
```

Comment:
Could you replace literal with a constexpr sizeof(bufRead) here and in the other similar places?